tests/unit/permissions_test.ts

```diff
@@ -20,54 +20,72 @@ Deno.test(function permissionInvalidNameSync() {
   }, TypeError);
 });
 
-Deno.test(async function permissionNetInvalidHost() {
-  await assertRejects(async () => {
-    await Deno.permissions.query({ name: "net", host: ":" });
-  }, URIError);
-});
-
-Deno.test(function permissionNetInvalidHostSync() {
-  assertThrows(() => {
-    Deno.permissions.querySync({ name: "net", host: ":" });
-  }, URIError);
-});
-
-Deno.test(async function permissionSysValidKind() {
-  await Deno.permissions.query({ name: "sys", kind: "loadavg" });
-  await Deno.permissions.query({ name: "sys", kind: "osRelease" });
-  await Deno.permissions.query({ name: "sys", kind: "osUptime" });
-  await Deno.permissions.query({ name: "sys", kind: "networkInterfaces" });
-  await Deno.permissions.query({ name: "sys", kind: "systemMemoryInfo" });
-  await Deno.permissions.query({ name: "sys", kind: "hostname" });
-  await Deno.permissions.query({ name: "sys", kind: "uid" });
-  await Deno.permissions.query({ name: "sys", kind: "gid" });
-  await Deno.permissions.query({ name: "sys", kind: "cpus" });
-});
-
-Deno.test(function permissionSysValidKindSync() {
-  Deno.permissions.querySync({ name: "sys", kind: "loadavg" });
-  Deno.permissions.querySync({ name: "sys", kind: "osRelease" });
-  Deno.permissions.querySync({ name: "sys", kind: "networkInterfaces" });
-  Deno.permissions.querySync({ name: "sys", kind: "systemMemoryInfo" });
-  Deno.permissions.querySync({ name: "sys", kind: "hostname" });
-  Deno.permissions.querySync({ name: "sys", kind: "uid" });
-  Deno.permissions.querySync({ name: "sys", kind: "gid" });
-  Deno.permissions.querySync({ name: "sys", kind: "cpus" });
-});
-
-Deno.test(async function permissionSysInvalidKind() {
-  await assertRejects(async () => {
-    // deno-lint-ignore no-explicit-any
-    await Deno.permissions.query({ name: "sys", kind: "abc" as any });
-  }, TypeError);
-});
-
-Deno.test(function permissionSysInvalidKindSync() {
-  assertThrows(() => {
-    // deno-lint-ignore no-explicit-any
-    Deno.permissions.querySync({ name: "sys", kind: "abc" as any });
-  }, TypeError);
-});
+Deno.test(
+  { permissions: { net: [] } },
```

Comment:
This change was necessary because it wasn't bothering to parse the host anymore when having full --allow-net permissions.